src/components/ResultsDisplay/index.jsx

```diff
@@ -0,0 +1,58 @@
+import RouteCard from "../RouteCard/index.jsx";
+import {useEffect, useState} from "react";
+import PlanRouteButton from "../PlanRouteButton/index.jsx";
+import NoRouteCard from "../NoRouteCard/index.jsx";
+import InvalidInputCard from "../InvalidInputCard/index.jsx";
+import {Link, Routes, Route} from "react-router-dom";
+import ClearRouteButton from "../ClearRouteButton/index.jsx";
+
+function ResultsDisplay({selected, destinationSelected}) {
+
+
+    const [journey, setJourney] = useState([])
+    const [boolean, setBoolean] = useState(false)
+    const [status, setStatus] = useState(0)
+
+    let originCode
+    let destinationCode
+
+    if (selected && destinationSelected) {
+        originCode = selected
+        destinationCode = destinationSelected
+    }
+
+    const fetchJourney = async () => {
+        const response = await fetch(`http://localhost:3000/journeys?origin=${originCode}&destination=${destinationCode}`)
+        setStatus(response.status)
+        let journeysArray
+        journeysArray = response.status == 200 ? await response.json() : []
+        setJourney(journeysArray.summary)
+    }
+
+    useEffect(() => {
+        fetchJourney()
+    }, [boolean]);
+
+    return (
+        <div className='container grid-cols-1  m5'>
+            <div className='flex justify-center gap-3 mb-5'>
+                <Link to={'/journeys'}>
+                    <PlanRouteButton boolean={boolean} setBoolean={setBoolean}/>
+                </Link>
+                <Link to={'/'}>
+                    <ClearRouteButton/>
+                </Link>
+            </div>
+            <Routes>
+                <Route path={'/journeys'} element={
```

Comment:
Is this the only route you're using? If so, why use BrowserRouter at all?

Reply:
got a few questions about this